### ISSUE TITLE
Failing Test

### ISSUE THREAD

**Issue by gdkrmr** [chengchingwen/Transformers.jl#35]
On current Julia master and Transformers master I have a failing test:

```
[ Info: Test BASIC
Basic: Test Failed at /home/gkraemer/.julia/dev/Transformers/test/test_basic.jl:10
  Expression: (p(x))[:, :, 2] == c(x[:, :, 2])
   Evaluated: Float32[0.027287507 0.16296661 … 0.2492562 0.4740091; 0.05988739 0.42276305 … 0.2984651 0.15890877; … ; 0.8943589 0.07297276 … 0.21390636 0.12822512; 0.0018252898 0.32664314 … 0.16835916 0.13867101] == Float32[0.027287507 0.16296661 … 0.2492562 0.47400913; 0.05988739 0.42276305 … 0.2984651 0.15890875; … ; 0.8943589 0.07297276 … 0.21390636 0.12822512; 0.0018252898 0.32664314 … 0.16835916 0.138671]
```

```julia
julia> using Transformers
julia> using Flux
julia> d = Flux.Dense(10, 5)
Dense(10, 5)
julia> c = Flux.Chain(d, Flux.softmax)
Chain(Dense(10, 5), softmax)
julia> p = Transformers.Positionwise(d, Flux.softmax)
Positionwise(Dense(10, 5), softmax)
julia> x = randn(10, 5, 3)
julia> p(x)[:, :, 2] .- c(x[:, :, 2])
5×5 Matrix{Float32}:
 0.0  0.0  0.0  0.0  -7.45058f-9
 0.0  0.0  0.0  0.0  -1.49012f-8
 0.0  0.0  0.0  0.0   0.0
 0.0  0.0  0.0  0.0  -2.79397f-9
 0.0  0.0  0.0  0.0   0.0
```

Not sure if these should be identical or if you can change the test from `==` to `≈`

**Comment by chengchingwen** [comment]
In theory they should be identical since `Positionwise` is just doing `reshape`, `Chain`, and then `reshape`, but I think some numerical error might be acceptable. If it 's the case, it should be ok to replace `==` with `≈`. 

I haven't test anything on Julia master. Correct me if I'm wrong but IIRC Flux (or Zygote?) also failed on Julia 1.6+. We will wait until all of the dependencies work correctly.

**Comment by chengchingwen** [comment]
fix with v0.1.8
